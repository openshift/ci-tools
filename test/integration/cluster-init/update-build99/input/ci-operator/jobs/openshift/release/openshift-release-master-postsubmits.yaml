postsubmits:
  openshift/release:
    - agent: kubernetes
      branches:
        - master
      cluster: app.ci
      decorate: true
      labels:
        ci.openshift.io/build-farm: build99
        ci.openshift.io/generator: cluster-init
        ci.openshift.io/role: infra
      max_concurrency: 1
      name: branch-ci-openshift-release-master-build99-apply
      spec:
        containers:
          - args:
              - --config-dir=clusters/build-clusters/build99
              - --as=
              - --kubeconfig=/etc/build-farm-credentials/kubeconfig
              - --confirm=true
              - --extra-arg=true
            command:
              - applyconfig
            image: registry.ci.openshift.org/ci/applyconfig:oldest
            imagePullPolicy: Always
            name: ""
            resources:
              requests:
                cpu: 1m
            volumeMounts:
              - mountPath: /etc/build-farm-credentials
                name: build-farm-creds
                readOnly: true
        serviceAccountName: config-updater
        volumes:
          - name: build-farm-credentials
            secret:
              items:
                - key: sa.config-updater.build99.config
                  path: kubeconfig
              secretName: config-updater
