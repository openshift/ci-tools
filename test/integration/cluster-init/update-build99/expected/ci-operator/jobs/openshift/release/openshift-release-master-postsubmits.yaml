postsubmits:
  openshift/release:
  - agent: kubernetes
    branches:
    - ^master$
    cluster: app.ci
    decorate: true
    labels:
      ci.openshift.io/build-farm: build99
      ci.openshift.io/generator: cluster-init
      ci.openshift.io/role: infra
    max_concurrency: 1
    name: branch-ci-openshift-release-master-build99-apply
    spec:
      containers:
      - args:
        - --config-dir=clusters/build-clusters/build99
        - --as=
        - --kubeconfig=/etc/build-farm-credentials/kubeconfig
        - --confirm=true
        command:
        - applyconfig
        env:
        - name: build99_id
          valueFrom:
            secretKeyRef:
              key: build99-id
              name: build99-dex-oidc
        image: registry.ci.openshift.org/ci/applyconfig:latest
        imagePullPolicy: Always
        name: ""
        resources:
          requests:
            cpu: 10m
        volumeMounts:
        - mountPath: /etc/build-farm-credentials
          name: build-farm-credentials
          readOnly: true
      serviceAccountName: config-updater
      volumes:
      - name: build-farm-credentials
        secret:
          items:
          - key: sa.config-updater.build99.config
            path: kubeconfig
          secretName: config-updater
